tests/Integration/Mapper/Fixtures/ObjectWithMappedVariousPropertyScope.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Tempest\Integration\Mapper\Fixtures;
+
+use Tempest\Mapper\Attributes\MapTo;
+
+final class ObjectWithMappedVariousPropertyScope
+{
+    public function __construct(
+        #[MapTo('private')] // @phpstan-ignore-line property.onlyWritten
```

Comment:
We actually don't need support for private and protected properties. One of the conventions with our mapper is that it only deals with public properties.